config/config.go

```diff
@@ -317,8 +317,8 @@ func LoadConf(confPath string) (ConfYaml, error) {
 
 	// Huawei
 	conf.Huawei.Enabled = viper.GetBool("huawei.enabled")
-	conf.Huawei.APIKey = viper.GetString("huawei.apikey")
-	conf.Huawei.APPId = viper.GetString("huawei.appid")
+	conf.Huawei.AppSecret = viper.GetString("huawei.appsecret")
+	conf.Huawei.AppId = viper.GetString("huawei.appid")
```

Comment:
Please fix the lint error https://cloud.drone.io/appleboy/gorush/503/1/4

Reply:
Hi, I fixed the errors for **AppID**, but **`make lint`** now throws different error. When I try `make lint` locally it doesn't show any errors. Can you check it out?